tmlibrary/tmlib/tools/classification.py

```diff
@@ -61,7 +67,9 @@ def process_request(self, submission_id, payload):
                 "options": {
                     "method": str,
                     "n_fold_cv": int
-                }
+                },
+                "task": str (either 'classification' or 'saveLabels'),
```

Comment:
for consistency I don't think camelCase is used at all in TissueMAPS, Therefore `saveLabels` should be `save_labels` throughout